src/burst2safe/utils.py

```diff
@@ -318,11 +319,10 @@ def reparse_args(args: Namespace, tool: str) -> Namespace:
             args.swaths = [swath.upper() for swath in args.swaths]
 
         if args.extent:
-            try:
+            if not (len(args.extent) == 4 or len(args.extent) == 1):
+                raise ValueError('--extent cannot be interpreted as a bounding box or geometry file.')
```

Comment:
I think this error reads a bit better like this:

```suggestion
                raise ValueError('The argument provided to --extent could not be interpreted as a bounding box (W S E N in lat/lon) or a geometry file.')
```